source/includes/administration/_service_connections.md

```diff
@@ -59,3 +59,112 @@ Attributes | &nbsp;
+
+### Retrieve connection policy descriptors
+
+`GET /services/connections/:id/policies/descriptors`
+
+```shell
+# Retrieve usage summary in CSV
+curl "https://cloudmc_endpoint/v1/services/connections/03bc22bd-adc4-46b8-988d-afddc24c0cb5/policies/descriptors?section=trials" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+Query Parameters | &nbsp;
+---------- | -----
+`section`<br/>*string* | The name of the policy section to load. Only the policy section matching the section name provided will return all required FormElements and the connection entity.
+
+```json
+{
+  "data":[{
+    "name": "general",
+    "label": "general",
+    "optional": false,
+  },{
+    "name": "trials",
+    "formElements": [{
+      "label": "cloudstack.service_configuration.policies.trials.network_config_for_trial_env.label",
+      "descriptionLabel": "cloudstack.service_configuration.policies.trials.network_config_for_trial_env.description",
+      "interpolation": {},
+      "options": [{
+        "value": "isolatedNetwork",
+        "type": "option",
+        "name": "cloudstack.service_configuration.policies.trials.network_config_for_trial_env.options.isolated_network",
+        "is18n": true
+      }, {
+        "value": "singleNetwork",
+        "type": "option",
+        "name": "cloudstack.service_configuration.policies.trials.network_config_for_trial_env.options.single_subnet_vpc",
+        "is18n": true
+      }],
+      "disabled": false,
+      "required": true,
+      "field": "trials:network_config_for_trial_envs",
+      "reloadOnChange": false,
+      "sectionsToReload": [],
+      "type": "select"
+    }],
+    "entity": {
+      "id": "7486dfbf-740d-49b1-ba68-6e4b0070ebbb",
+      "name": "Google Cloud Connection",
+      "policies": {
+        "version": "1.0",
+        "cacheEnabled": "false",
+        "regions": "us-east,eu-west"
+      }
+    }
+  }]
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`name`<br/>*string* | The name of the policy section
+`label`<br/>*string* | The label key for the policy section name.
+`formElements`<br/>*Array* | The FormElements returned by the policy section. Form elements are only returned for a given section if the query param `section` matches the name of a policy section.  
+`optional`<br/>*boolean* | Specifies if the policy section is required or not.
+`entity`<br/> *Object* | The service connection entity which includes a string to string map of the ServiceConnectionPolicy::name to ServiceConnectionPolicy::value
+
+
+<!-------------------- UPDATE CONNECTION POLICIES -------------------->
+
+### Update connection policies
+
+`PUT /services/connections/:id/policies`
+
+```shell
+curl -X PUT \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "[{"name": "serviceVersion", "value": "1.1"}, {"name": "cacheEnabled", "value": "true"}]" \
+   "https://cloudmc_endpoint/v1/services/compute-on/test_area/ingressrules"
```

Comment:
Wrong url, it should be: `https://cloudmc_endpoint/v1/services/connections/03bc22bd-adc4-46b8-988d-afddc24c0cb5/policies`